app/models/purl.rb

```diff
@@ -20,6 +20,29 @@ class Purl < ActiveRecord::Base
   }
 
   ##
+  # Return true targets with always values only if the object is not deleted in
+  # purl mount
+  # @return [Array]
+  def computed_true_targets
```

Comment:
I like the idea of putting the logic in true_targets if that is the normal scenario of what is always going to be returned.  Even though it has the potential to confuse a consumer of the class (maybe they expected all), it reduces the risk of someone inadvertently returning the targets when they shouldn't (which seems like the normal case).  

Also, doesn't the update to the mark_deleted method method below means they should be cleaned up from now on?